commands/help.js

```diff
@@ -1,13 +0,0 @@
-const Command = require('../library/command');
-
-/**
- * Displays a list of commands available
- * 
- * @class HelpCommand
- * @extends {Command}
- */
-
-class HelpCommand extends Command {
```

Comment:
What kind of monster would commit broken code to the repo

Reply:
I was wondering why those were there, I thought maybe it was for a grander design.